.schema/config.schema.json

```diff
@@ -246,9 +246,14 @@
       "properties": {
         "to": {
           "title": "Redirect to",
-          "description": "Set the redirect target. Must be a http/https URL.",
+          "description": "Set the redirect target. Can either be a http/https URL, or a relative URL.",
           "type": "string",
-          "format": "uri"
+          "format": "uri-reference",
+          "examples": [
+            "https://my-app.com/dashboard",
+            "https://my-app.com/dashboard",
```

Comment:
```suggestion
```
Remove duplicate?

Reply:
Replaced with http in the examples, and added an additional test for both 301 and 302 to include both HTTP and HTTPS uris.